src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Azure/Functions/IFunctionInvoker.cs

```diff
@@ -0,0 +1,14 @@
+// <copyright file="IFunctionInvoker.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.Azure.Functions
+{
+    /// <summary>
+    /// For duck typing
+    /// </summary>
+    public interface IFunctionInvoker
```

Comment:
What do we use an empty interface for?

Reply:
Some of these were for testing and figuring out what was available, I'll remove any interfaces or properties we do not use.